crates/client/mempool/src/genesis/mod.rs

```diff
@@ -0,0 +1,471 @@
+use blockifier::abi::abi_utils::get_storage_var_address;
+use dc_block_import::{UnverifiedFullBlock, UnverifiedHeader};
+use dp_block::header::GasPrices;
+use dp_chain_config::ChainConfig;
+use dp_convert::ToFelt;
+use dp_state_update::{ContractStorageDiffItem, StateDiff, StorageEntry};
+use starknet_api::{core::ContractAddress, state::StorageKey};
+use starknet_signers::SigningKey;
+use starknet_types_core::felt::Felt;
+use std::{collections::HashMap, fmt, time::SystemTime};
+
+mod balances;
+mod classes;
+mod contracts;
+mod entrypoint;
+
+pub use balances::*;
+pub use classes::*;
+pub use contracts::*;
+pub use entrypoint::*;
+
+// 1 ETH = 1e18 WEI
+const ETH_WEI_DECIMALS: u128 = 1_000_000_000_000_000_000;
+// 1 STRK = 1e18 FRI
+const STRK_FRI_DECIMALS: u128 = 1_000_000_000_000_000_000;
+
+#[derive(Debug, Clone, Default)]
+pub struct StorageDiffs(HashMap<ContractAddress, HashMap<StorageKey, Felt>>);
+impl StorageDiffs {
+    pub fn contract_mut(&mut self, contract_address: ContractAddress) -> &mut HashMap<StorageKey, Felt> {
+        self.0.entry(contract_address).or_default()
+    }
+
+    pub fn as_state_diff(&self) -> Vec<ContractStorageDiffItem> {
+        self.0
+            .iter()
+            .map(|(contract, map)| ContractStorageDiffItem {
+                address: contract.to_felt(),
+                storage_entries: map.iter().map(|(key, &value)| StorageEntry { key: key.to_felt(), value }).collect(),
+            })
+            .collect()
+    }
+}
+
+// We allow ourselves to lie about the contract_address. This is because we want the UDC and the two ERC20 contracts to have well known addresses on every chain.
+
+/// Universal Deployer Contract.
+const UDC_CLASS_DEFINITION: &[u8] =
+    include_bytes!("../../../../../cairo/target/dev/madara_contracts_UniversalDeployer.contract_class.json");
```

Comment:
let's define a proper `CAIRO_CONTRACTS_PATH` constant

Reply:
i tried that and `concat!` to concat the path did not work with include_bytes

Reply:
that's devastatingly sad